vms/platformvm/service.go

```diff
@@ -948,53 +1004,64 @@ func (s *Service) GetL1Validator(r *http.Request, args *GetL1ValidatorArgs, repl
 		return fmt.Errorf("fetching L1 validator %q failed: %w", args.ValidationID, err)
 	}
 
+	ctx := r.Context()
+	height, err := s.vm.GetCurrentHeight(ctx)
+	if err != nil {
+		return fmt.Errorf("failed getting current height: %w", err)
+	}
+	apiVdr, err := s.convertL1ValidatorToAPI(l1Validator)
+	if err != nil {
+		return fmt.Errorf("failed converting L1 validator to API: %w", err)
+	}
+
+	reply.APIL1Validator = apiVdr
+	reply.SubnetID = l1Validator.SubnetID
+	reply.Height = avajson.Uint64(height)
```

Comment:
curious - what's the point of `avajson.Uint64` if it's just a uint64?

Reply:
it's to format numbers with double-quotes:  `"123"`